src/renderer/Renderer.js

```diff
@@ -268,13 +271,15 @@ export default class Renderer {
         }
         dataframes.map(dataframe => styleDataframe(dataframe, dataframe.texFilter, viz.filterMetaShader, viz.filter));
 
+        // Final drawing (to screen). In the case of lines / polygons, there is an extra step (for antialiasing)
         gl.blendFunc(gl.ONE, gl.ONE_MINUS_SRC_ALPHA);
         gl.enable(gl.BLEND);
 
-        gl.bindFramebuffer(gl.FRAMEBUFFER, null);
-        gl.viewport(0, 0, gl.drawingBufferWidth, gl.drawingBufferHeight);
-
-        if (renderLayer.type !== 'point') {
+        if (renderLayer.type === 'point') {
```

Comment:
After https://github.com/CartoDB/carto-vl/pull/1146 gets merged, I guess we need to update this one with the master branch and reuse the constants.

Reply:
Great!, I'm gonna merge that one first and then adapt this!